scripts/bootstrap.sh.in

```diff
@@ -53,6 +53,12 @@ determine_os() {
         # shellcheck disable=SC1091
         . /etc/os-release
         OS=$ID
+	[[ $OS = rhel ]] && OS=redhat
+	if [[ $OS =~ ^(redhat|centos)$ ]]; then
+            OS_FAMILY=redhat
+        elif [[ $OS =~ ^(debian|ubuntu)$ ]]; then
+            OS_FAMILY=debian
+	fi
```

Comment:
Look like you have some indentation issue here

Reply:
No, it's intended, I like it this way ! :p